src/App.js

```diff
@@ -24,6 +25,7 @@ function App() {
             ))
           }
         </a>
+        <p>do you even <strong>G</strong>it</p>
```

Comment:
strong tag -> maybe preferable to use span + CSS?

Reply:
im on it